vispy/visuals/image.py

```diff
@@ -87,6 +87,21 @@
         return texture2D($texture, texcoord);
     }"""
 
+import array, struct
+
+def floatToFixed(f):
+    str = array.array('f', f.ravel()).tostring()
+    print(len(str))
+    size = f.size * 4
+    fmt = "!%dB" % size
+    print(fmt, size, f.shape + (4,))
```

Comment:
This would be very nice, I tried with numpy but got no solution, so I took this one which worked for testing.